packages/repository/src/__tests__/unit/model/model.unit.ts

```diff
@@ -444,6 +444,31 @@ describe('model', () => {
     });
   });
 
+  it('hide a propety with PropertyDefinition', () => {
+    class UserHidden extends Entity {
```

Comment:
`UserHidden` -> `User`

Reply:
already exists a class called User to avoid overwrite the existed one i used that name

Reply:
Can we use a class/model name like `Customer`?

Personally, I don't mind `UserHidden` too much, since it's scoped to this single test only.